RxSwift/Schedulers/ConcurrentDispatchQueueScheduler.swift

```diff
@@ -44,38 +49,35 @@ public class ConcurrentDispatchQueueScheduler: SchedulerType {
         )
     }
 
-    /**
-    Schedules an action to be executed immediately.
-    
-    - parameter state: State passed to the action to be executed.
-    - parameter action: Action to be executed.
-    - returns: The disposable object used to cancel the scheduled action (best effort).
-    */
+    /// Schedules an action to be executed immediately.
```

Comment:
Any reason you changed all comment blocks from 
```
/**

*/
```
To
``` 
///
///
///
```

Please leave the existing style as that is the style used in all source files in the framework.